debug/testlib.py

```diff
@@ -276,11 +277,11 @@ def __init__(self, address):
 class Gdb(object):
     logfile = tempfile.NamedTemporaryFile(prefix="gdb", suffix=".log")
     logname = logfile.name
+    print "GDB Temporary file: %s" % logname
```

Comment:
That's right. I'm happy to call them temporary, as long as it's consistent in both print statements.